docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -105,6 +105,61 @@ include::../../../../extensions/agroal/deployment/src/test/resources/application
 
 There are other configuration options, detailed below.
 
+=== JDBC URL configurations
+
+Each of the supported databases contains different JDBC URL configuration options.
+Going into each of those options is beyond the scope of this document, but it give an overview of each database URL and link to the official documentation.
+
+==== H2
+
+`jdbc:h2:{ {.|mem:}[name] | [file:]fileName | {tcp|ssl}:[//]server[:port][,server2[:port]]/name }[;key=value...]`
+
+Example:: `jdbc:h2:tcp://localhost/~/test`, `jdbc:h2:mem:myDB`
+
+H2 is an embedded database.
+It can run as a server, based on a file, or live completely in memory.
+All of these options are available as listed above.
+You can find more information at the https://h2database.com/html/features.html?highlight=url&search=url#database_url[official documentation].
+
+==== PostgreSQL
+
+PostgreSQL only runs as a server, as do the rest of the databases below.
+As such, you must specify connection details, or use the defaults.
+
+`jdbc:postgresql:[//][host][:port][/database][?key=value...]`
+
+Example:: `jdbc:postgresql://localhost/test`
+
+Defaults for the different parts are as follows:
+
+`host`:: localhost
+`port`:: 5432
+`database`:: same name as the username
+
+WARNING: There is no default username, it must be specified as a property.
+
+The https://jdbc.postgresql.org/documentation/head/connect.html[official docs] go into more detail and list optional parameters as well.
```

Comment:
missed it the first time but same here: let's go for `official documentation`. Then s/go/goes/ and s/list/lists/.